brewcop.py

```diff
@@ -217,64 +205,238 @@ class Brewcop:
                       `"""----"""`
 '''
 
-    def __init__(self):
-        try:
-            self.scale = Scale()
-        except:
-            self.scale = NoScale()
-        self._indicator = urwid.Text("", align="right")
-        self._meter = urwid.BigText("", urwid.Thin6x6Font())
-        self._online = False
-        self.banner = urwid.Text(self.banner_msg, align="left")
-        self.status = urwid.Text(self.off_msg, align="center")
-        self.pbar = Progress_mL("pb_todo", "pb_done", 0, self.pot_capacity_g)
-        self.background = self.init_background(self.coffee_cup)
-        self.meterbody = self.init_meterbody(self._meter, self.background)
-        self.header = urwid.Columns([self.banner, self._indicator], 2)
-        self.layout = urwid.Frame(
-            header=self.header, body=self.meterbody, footer=self.status
-        )
-        self.main_loop = urwid.MainLoop(
-            self.layout, self.palette, unhandled_input=self.handle_input
-        )
+    def __init__(self, pot_capacity_mL=100):
+        """header"""
```

Comment:
Minor nit: I initially thought this was the multi-line string documenting the entire function.  It may be better to comment these with `#`.